### ISSUE TITLE
Feature request - ogurictl reload

### ISSUE THREAD

**Issue by mnussbaum** [vilhalmer/oguri#36]
Hi there, thanks so much for oguri, it's a great tool!

I'm opening this issue to express interest in a way to have oguri re-read its config file and seamlessly apply any changes. My use case is that I want to have my wallpaper change depending on the time of day. I have a system that re-writes configs for color schemes and other theme elements depending on the time of day, and I'd like to have it re-write my oguri config and then reload oguri to show a new wallpaper.

I could use ogurictl now to directly load a path to a new background image, but that would require running a different ogurictl command depending on the time of day which doesn't quite work for me.

Thanks for your consideration!

**Comment by vilhalmer** [comment]
Yep, I want this too. I'm still pondering how to make the IPC protocol do it.
